tidb-cloud/changefeed-sink-to-mysql.md

```diff
@@ -53,13 +53,19 @@ If your MySQL service is in a GCP VPC that has no public internet access, take t
 
     You must add [the CIDR of the region where your TiDB Cloud cluster is located](/tidb-cloud/set-up-vpc-peering-connections.md#prerequisite-set-a-project-cidr) to the ingress firewall rules. Doing so allows the traffic to flow from your TiDB Cluster to the MySQL endpoint. 
 
-### Full load data
+### Create a target table in MySQL
+
+If you decide no to load the existing data, you need to create a target table in MySQL with the same name as that of the source table in TiDB in advance. Otherwise, the data will not be replicated.
+
```

Comment:
Delete the  following sentence
### Create a target table in MySQL

If you decide no to load the existing data, you need to create a target table in MySQL with the same name as that of the source table in TiDB in advance. Otherwise, the data will not be replicated.